runtime/reference/vscode.md

```diff
@@ -0,0 +1,510 @@
+  `deno test` when activating a test code lens. _string array, default
+  `["--allow-all"]`_
+
+### Mixed-Deno projects
+
+While you can use this feature to enable mixed-Deno projects, you might want to
+consider
+[partially Deno enabling a workspace](#partially-deno-enabling-a-workspace). But
+with this feature, you can have a mixed Deno project, where some of the
+workspace folders are Deno enabled and some are not. This is useful when
+creating a project that might have a front-end component, where you want a
+different configuration for that front end code.
+
+In order to support this, you would create a new workspace (or add a folder to
+an existing workspace) and in the settings configure one of the folders to have
+`deno.enable` set to `true` and one set to `false`. Once you save the workspace
+configuration, you notice that the Deno language server only applies diagnostics
+to the enabled folders, while the other folder will use the built-in TypeScript
+compiler of VS Code to supply diagnostics for TypeScript and JavaScript files.
+
+## Using a development container
+
+Using a
+[development container](https://code.visualstudio.com/docs/remote/containers)
+with VS Code is a great way to have an isolated development environment without
+having to worry about having to install the Deno CLI on your local system.
+
+To use development containers, you need to have a few
+[prerequisites installed](https://code.visualstudio.com/docs/remote/containers#_installation):
+
+- Docker Desktop
+- Visual Studio Code or Visual Studio Code Insiders
+- [Remote Development extension pack](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack)
+
+The way a development container is configured is by having a `.devcontainer`
+folder as part of the workspace with configuration information in the folder. If
+you are opening a project that already contains a dev container with Deno, you
+will be prompted to build the dev container and access the project under that.
+Everything should "just work".
+
+If you have an existing Deno project that you would like to add dev container
+support to, you will want to execute the command _Remote-Containers: Add
+Development Container Configuration Files..._ in the command palette and then
+choose **Show All Definitions...** and then search for the `Deno` definition.
+This will setup a baseline `.devcontainer` configuration for you that will
+install the latest version of the Deno CLI in the container.
+
+Once added, VS Code will prompt if you want to open the project in a dev
+container. If you choose to, VS Code will build the development container and
+re-open the workspace using the development container, which will have the Deno
+CLI and the `vscode_deno` extension installed in it.
+
+## Troubleshooting
+
+The following sections cover challenges you might face when using the extension
+and try to give likely causes.
```

Comment:
This is a great vscode feature, but not sure if we need to describe it in our docs if it's a "just works" with Deno thing.